modules/ibi-generate-seed-image.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+// * edge_computing/ztp-image-based-upgrade.adoc
+// * edge_computing/factory-image-based-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The Operator checks for required system configurations, performs any necessary system cleanup before generating the seed image, and launches the image generation. The seed image generation includes the following tasks:
+
+* Stopping cluster operators
+* Preparing the seed image configuration
+* Generating and pushing the seed image to the image repository specified in the `SeedGenerator` CR
+* Restoring cluster operators
+* Expiring seed cluster certificates
+* Generating new certificates for the seed cluster
+* Restoring and updates the `SeedGenerator` CR on the seed cluster
+
+[NOTE]
+====
+The generated seed image does not include any site-specific data.
+
+To avoid including any {rh-rhacm-first} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons for seed image generation.
+====
+
+.Prerequisites
+
+* You have deployed a managed {sno} cluster with a DU profile.
+* Install the {lcao} in the {sno} cluster.
+* Log in as a user with `cluster-admin` privileges.
+* Review the required versions of software components in the section _Software prerequisites for an image-based installation_.
+* If you use {rh-rhacm-first}, review the hub cluster guidelines in the section _Hub cluster guidelines_.
+* You created an extra partition in the seed cluster. See _Configuring a shared container directory between `ostree` stateroots_ for further information.
+
+.Procedure
+
+. Detach the cluster from the hub to delete any cluster-specific resources from the seed cluster that must not be in the seed image:
```

Comment:
```suggestion
We assume here that the cluster is a spoke, but nowhere else we mentioned that the cluster is a spoke.
I think we should something like: If the cluster is a spoke, detach it from the hub...
```

Reply:
I meant that the cluster doesn't have to be spoke.

Reply:
I see what you mean, updated to `If the cluster is a managed cluster, detach the cluster ...`